clients/drcachesim/drpt2trace/pt2ir.h

```diff
@@ -359,13 +362,19 @@ class pt2ir_t {
      * @param pt_data The PT raw trace.
      * @param pt_data_size The size of PT raw trace.
      * @param drir The drir object.
+     * @param non_fatal_decode_error_count_out Pointer to the integer where the count
+     * non-fatal decode errors seen during conversion will be stored. Used only if
+     * allow_non_fatal_decode_errors was set to true in the init call. This is set
+     * only if we were still able to generate a converted trace, albeit with some
+     * PC discontinuities.
```

Comment:
And potentially some missing syscall traces?

Reply:
pt2ir doesn't make that decision, raw2trace does. That fact is documented at raw2trace_t::pt2ir_best_effort_.